packages/common/core-interfaces/Removing-IFluidRouter.md

```diff
@@ -46,18 +46,21 @@ The new way to do this is via the object's "entry point".
 Here it is on `IContainer`, returning an anonymous `FluidObject` - the application-specified root object:
```

Comment:
We should probably remove the part about request above.

Reply:
I think it's ok to leave the historical reason behind the request pattern here to give some devs clarity on things they should be looking for in their code

Reply:
That sounds fine. But we should at least mention that request is deprecated / removed.